tests/playbooks/integration_pytest_python3.yml

```diff
@@ -0,0 +1,100 @@
+# SPDX-License-Identifier: BSD-3-Clause
+---
+- name: Install deps for integration tests
+  hosts: all
+  vars:
+    - rpmdependencies:
+        - git
+        - python3-pip
+        - rsync
+
+  tasks:
+    - name: Install EPEL for RHEL and CentOS
+      # yamllint disable-line rule:line-length
+      command: "yum install -y https://dl.fedoraproject.org/pub/epel/epel-release-latest-8.noarch.rpm"
+      when: (ansible_facts['distribution'] == "CentOS") or
+            (ansible_facts['distribution'] == "RedHat")
+
+    - name: Install rpm dependencies
+      package:
+        state: present
+        name: "{{ rpmdependencies }}"
+
+    - name: Search for rsync path
+      command: which rsync
+
+    - name: Install Pytest
+      command: "pip3 install pytest"
+
+- name: Run Pytest tests
+  hosts: all
+  vars:
+    - rundir: /run/system-roles-test/
```

Comment:
Which task failed? The copy task for the tarball?

Reply:
The fail was on the copy. However, it is not happening now. I will remove the slash then.

Reply:
I see, this is a special behavior of the copy module: https://docs.ansible.com/ansible/latest/modules/copy_module.html#parameter-dest - it works now without the slash because the playbook creates the directory beforehand (using the modified copy task).